cprover_bindings/src/goto_program/expr.rs

```diff
@@ -131,7 +131,7 @@ pub enum ExprValue {
     StringConstant {
         s: InternedString,
     },
-    /// Struct initializer  
+    /// Struct initializer
```

Comment:
I'm all for whitespace cleanup, but can this perhaps go in a PR of its own?

Reply:
Sure... my IDE just does that automatically

Reply:
btw, this change will go away once I merge #2405